Documentation for ndd.DType (#6170)
NVIDIA/DALI: docs/dali_dynamic/api_reference.rst

^^^^^^^^
.. autofunction:: as_batch

Data types
----------

Those are the data type objects that DALI Dynamic uses to indicate the type of elements of Tensors and Batches. 

> They can typically be passed as `dtype` argument to request specific output type of the operator. 

Reviewer: ```suggestion
They are typically passed as `dtype` argument to request specific type of the tensor element. 
```
Rationale: `dtype` is also (and perhaps predominantly) passed to tensor/batch factories, not (just) operators.
